modules/OSINTFootprinting/ActiveRecon/commentssrc.py

```diff
@@ -55,7 +55,7 @@ def commentssrc(web):
         print(GR+' [*] Making the request...')
         req = requests.get(w).content
         print(C+' [!] Setting parse parameters...')
-        comments = re.findall('<!--(.*)-->',req)
+        comments = re.findall('<!--(.*)-->',str(req))
```

Comment:
There are 2 ways of approaching this:
- Either make the regex binary.
- Convert L56 to use the `.text` attribute instead of `.content`.

I say we use the 2nd option instead of explicitly converting bytes to str.